src/org/zaproxy/zap/extension/AllInOneNotes/NotesTableModel.java

```diff
@@ -0,0 +1,49 @@
+package org.zaproxy.zap.extension.AllInOneNotes;
```

Comment:
This should have the standard licenseheader: https://github.com/zaproxy/zap-admin/blob/master/docs/headers/license.java
(With `$YEAR` appropriately set obviously)

Reply:
added thank you!